### ISSUE TITLE
"Home" and "End" keys don't work in the search bar

### ISSUE THREAD

**Issue by kuba-orlik** [mdn/yari#11079]
### Summary

The "End" and "Home" keyboard keys do not move the cursor or selection while entering text in the search bar:

![image](https://github.com/mdn/yari/assets/2697916/aa99c2a0-4f9f-45ea-a3dd-3f705b6f95dc)

### URL

https://developer.mozilla.org/

### Reproduction steps

1. Go to MDN
2. Enter some text in the search bar
3. Press Home

### Expected behavior

The cursor should move to the beginning of the text

### Actual behavior

Nothing happens

### Device

Desktop

### Browser

Firefox

### Browser version

Pre-release (e.g. Beta, Nightly or Canary)

### Operating system

Linux

### Screenshot

[Screencast from 2024-05-06 18-27-37.webm](https://github.com/mdn/yari/assets/2697916/124f3620-f8a7-46ea-8b07-28a55faebdd2)

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by mirunacurtean** [comment]
@kuba-orlik Thank you for your contribution.  This is in line with to a [list of similiar accesibility issues when it comes to search](https://github.com/mdn/yari/issues/5650). 

One note, while Home and End keys cannot be used to navigate the search box input, they can be used to navigate the search results. 

https://github.com/mdn/yari/assets/53306802/69dc927d-ad60-4fab-b983-09da16640666

**Comment by caugner** [comment]
Thanks @mirunacurtean.

@kuba-orlik See [this comment](https://github.com/mdn/yari/issues/10074#issuecomment-1824608466) for our position on this, and an alternative. I'm closing this as won't fix.
